packages/tools/tests/unit/helpers/cs-field-editor-for-test.js

```diff
@@ -20,11 +24,21 @@ test('it maps to a field editor component', function(assert) {
   assert.equal(result, 'field-editors/test-field-type-editor');
 });
 
+test('it can do a custom field editor', function(assert) {
+  let result = csFieldEditorFor([new TestModel(), 'theFieldWithCustomEditor'], {});
```

Comment:
Should we give the default variant a name?

Reply:
Sure, it could be "default".